<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>764</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>764</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Mono</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>TFM - Tor Forensic Manager</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../.designer/img/Logo.png</normaloff>../../../../.designer/img/Logo.png</iconset>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Show help&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(243, 243, 243);
selection-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="appNameLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TOR FORENSIC MANAGER</string>
    </property>
   </widget>
   <widget class="QLabel" name="appVersionLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Version 1.0 Fco Javier 2022Â©</string>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>401</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/Logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="UEMLabel">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>490</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../.designer/img/UEM.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="date_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>300</width>
      <height>18</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Mono</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPrincipal">
    <property name="title">
     <string>Principal</string>
    </property>
    <addaction name="actionTor_info"/>
    <addaction name="actionTorrc"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFile_Forensic">
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open files in SQLite Browser&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="title">
     <string>File Forensic</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionManage_Files_Forensic"/>
    <addaction name="actionOpen_Web_Viewer"/>
   </widget>
   <widget class="QMenu" name="menuNodes_Info">
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Display node info&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="title">
     <string>Nodes Info</string>
    </property>
    <addaction name="actionShow_Nodes_Info"/>
   </widget>
   <widget class="QMenu" name="menuMore">
    <property name="title">
     <string>More</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionGitHub_Repository"/>
    <addaction name="actionMake_security_copy"/>
    <addaction name="actionOpen_security_copy"/>
   </widget>
   <addaction name="menuPrincipal"/>
   <addaction name="menuFile_Forensic"/>
   <addaction name="menuNodes_Info"/>
   <addaction name="menuMore"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionGitHub_Repository">
   <property name="text">
    <string>GitHub Repository</string>
   </property>
   <property name="toolTip">
    <string>GitHub Repository</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGeneral_Information">
   <property name="text">
    <string>General Information</string>
   </property>
  </action>
  <action name="actionUpdate_Information">
   <property name="text">
    <string>Update Information</string>
   </property>
  </action>
  <action name="actionTor_info">
   <property name="text">
    <string>Tor information</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionTorrc">
   <property name="text">
    <string>Torrc File</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionManage_Files_Forensic">
   <property name="text">
    <string>Manage</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionOpen_Web_Viewer">
   <property name="text">
    <string>Open Web Viewer</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionShow_Nodes_Info">
   <property name="text">
    <string>Show Nodes Info</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionMake_security_copy">
   <property name="text">
    <string>Make security copy</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_security_copy">
   <property name="text">
    <string>Open security copy</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Mono</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
